src/CarwowTheme/Filters.elm

```diff
@@ -124,21 +159,26 @@ getFilterGroupItemLabel item =
 
 {-| Placeholder
 -}
-filterGroup : List (FilterGroupItem msg) -> String -> List (Html.Html msg)
-filterGroup items label =
-    List.map (\item -> filterGroupItem item label) items
+filterGroup : List (FilterGroupItem msg) -> String -> String -> List (Html.Html msg)
+filterGroup items label filterPrefix =
+    List.map (\item -> filterGroupItem item label filterPrefix) items
 
+{-| Placeholder
+-}
+filterGroupWithExpander : List (FilterGroupItem msg) -> String -> String -> List (Html.Html msg)
+filterGroupWithExpander items label filterPrefix =
+    List.map (\item -> filterGroupItemWithExpander item label filterPrefix) items
 
 {-| Placeholder
 -}
-standardFilterView : String -> String -> List (FilterGroupItem msg) -> Html.Html msg
-standardFilterView label selectedIcon items =
+standardFilterView : String -> String -> List (FilterGroupItem msg) -> String -> Html.Html msg
+standardFilterView label selectedIcon items alignment =
     let
         itemsCount =
             List.length items
 
         content =
-            filterGroup items label
+            filterGroup items label ""
```

Comment:
This says to me that the `filterPrefix` should be a `Maybe String`, rather than a String which is sometimes empty...